Lib/glyphsLib/builder/font.py

```diff
@@ -82,11 +82,17 @@ def to_ufo_font_attributes(self, family_name):
 
         self.to_ufo_names(ufo, master, family_name)
         self.to_ufo_family_user_data(ufo)
+
+        ufo.lib[UFO2FT_FILTERS_KEY] = [
```

Comment:
What if the UFO has preexisting filters though?

Reply:
some Glyphs-specific filters (e.g. Transformations) are stored in the same lib key when exporting custom parameters. Depending on the order of these operations (i'm not sure exactly which one happens first) it may be that that list is non-empty. Better to avoid overwriting, try do a `ufo.lib.setdefault(UFO2FT_FILTERS_KEY, []).append(...)`